ui/components/app/confirm/title/title.tsx

```diff
@@ -0,0 +1,51 @@
+import React, { memo } from 'react';
+import { useSelector } from 'react-redux';
+import { TransactionType } from '@metamask/transaction-controller';
+import { Text } from '../../../component-library';
+import {
+  TextVariant,
+  TextAlign,
+  TextColor,
+} from '../../../../helpers/constants/design-system';
+import { useI18nContext } from '../../../../hooks/useI18nContext';
+import { currentConfirmationSelector } from '../../../../selectors';
+
+const typeToTitleTKey: Partial<Record<TransactionType, string>> = {
+  [TransactionType.personalSign]: 'confirmTitleSignature',
+};
+
+const typeToDescTKey: Partial<Record<TransactionType, string>> = {
+  [TransactionType.personalSign]: 'confirmTitleDescSignature',
+};
+
+export const ConfirmTitle: React.FC = memo(() => {
+  const t = useI18nContext();
+  const currentConfirmation = useSelector(currentConfirmationSelector);
+
+  if (!currentConfirmation) {
+    return null;
+  }
+
+  const title = t(typeToTitleTKey[currentConfirmation.type]);
+  const description = t(typeToDescTKey[currentConfirmation.type]);
+
+  return (
+    <>
+      <Text
+        variant={TextVariant.headingLg}
+        paddingTop={4}
+        paddingBottom={2}
+        textAlign={TextAlign.Center}
+      >
+        {title}
+      </Text>
+      <Text
+        paddingBottom={4}
+        color={TextColor.textAlternative}
+        textAlign={TextAlign.Center}
+      >
+        {description}
+      </Text>
+    </>
+  );
+});
```

Comment:
heads up, we might want to consider passing the title and description down to this component. this could get pretty messy down the road because we won't have a 1-to-1 mapping of TransactionTypes to title & description.

If we do turn this back to a presentational component, please feel free to refer to 
https://github.com/MetaMask/metamask-extension/pull/21960/commits/5433b9731f71f8533f66a61ffbe665b510bdcc5c

cc: @jpuri